src/app/content/highlights/components/ShowMyHighlightsStyles.tsx

```diff
@@ -126,6 +143,106 @@ export const HighlightContentWrapper = styled.div`
   }}
 `;
 
+// tslint:disable-next-line:variable-name
+export const HighlightToggleEdit = styled.div`
+  .highlight-toggle-edit {
+    position: absolute;
+    width: 150px;
+    top: 1.2rem;
+    right: 0;
+  }
+
+  ${DropdownFocusWrapper} {
+    svg {
+      color: ${theme.color.neutral.darkest};
+    }
+  }
+
+  ${ColorPicker} {
+    label {
+      margin: 0.6rem;
+      width: 1.6rem;
+      height: 1.6rem;
+
+      svg {
+        width: 1.2rem;
+        height: 1.2rem;
+      }
+    }
+  }
+
+  ${DropdownList} {
+    padding: 0;
+
+    li {
+      display: flex;
+
+      a {
+        width: 100%;
+
+        svg {
+          width: 15px;
+          height: 15px;
+          margin-right: 10px;
+          color: ${theme.color.text.default};
+        }
+      }
+    }
+  }
+
+  ${MenuToggle} {
+    float: right;
+    margin-right: 0.2rem;
+
+    &:hover {
+      svg {
+        color: ${theme.color.secondary.lightGray.darkest};
+      }
+    }
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const HighlightEditButtons = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 0 8px;
+
+  button:first-child {
+    margin-right: 8px;
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const HighlightToggleEditContent = styled.div`
+  z-index: 2;
+  border: 1px solid ${theme.color.neutral.formBorder};
+  background-color: ${theme.color.neutral.formBackground};
+`;
+
+// tslint:disable-next-line:variable-name
+export const HighlightDeleteWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
```

Comment:
i usually specify all of top/bottom/left/right to 0 instead of with/height 100 to take up the whole container, do you know if there is a difference in behavior? i think this is fine i'm just curious.

Reply:
I don't know, I'm just used to using this way